src/utils/summary.cpp

```diff
@@ -541,20 +564,27 @@ DataType lbann_summary::local_max(const Mat& mat) const {
   const El::Int ldim = mat.LDim();
   const DataType * __restrict__ mat_buf = mat.LockedBuffer();
   auto max = std::numeric_limits<DataType>::min();
+  int nthreads = omp_get_num_threads();
+  std::vector<EvalType> local_max(nthreads, std::numeric_limits<DataType>::min());
```

Comment:
This needs to be `DataType`, not `EvalType`, or it will fail to compile.

Reply:
Fixed.